src/layer/poi.js

```diff
@@ -84,6 +84,70 @@ var iconDefs = {
     color: Color.poi.infrastructure,
     description: "Police station",
   },
+  pow_buddhist: {
+    classes: {
+      place_of_worship: ["buddhist"],
+    },
+    sprite: "poi_pow_buddhist",
+    color: Color.poi.infrastructure,
+    description: "Buddhist place of worship",
+  },
+  pow_christian: {
+    classes: {
+      place_of_worship: ["christian"],
+    },
+    sprite: "poi_pow_christian",
+    color: Color.poi.infrastructure,
+    description: "Christian place of worship",
```

Comment:
id-tagging-schema calls this tagging combination simply “[Christian church](https://github.com/openstreetmap/id-tagging-schema/blob/c0bfb2812c1d0ae94cc05fa0567002c787296551/data/presets/amenity/place_of_worship/christian.json#L46)”, and there are other religion-specific descriptions for other `religion` values.

Reply:
I'm open to this, and considered it for the initial PR, though I worried it gets into complicated terminology issues quickly. For instance, not all `amenity=place_of_worship` + `religion=christian` are churches (like LDS Temples or even [holy wells](https://wiki.openstreetmap.org/wiki/Tag%3Aplace_of_worship%3Dholy_well)), some sects of Judaism call their houses of worship 'temples' or 'shuls' instead of synagogues, what iD labels as "Sikh temples" are actually called [gurdwaras](https://en.wikipedia.org/wiki/Gurdwara), etc. But maybe these are small potatoes compared to the improvement in how understandable the legend would be.

Reply:
That’s true, your point probably matters more in the context of a public-facing map like Americana than something internal to OSM like iD. The diplomatic “place of worship” terminology isn’t inaccurate, just wordy, but we have enough room for it.